src/openms/source/SIMULATION/RawMSSignalSimulation.cpp

```diff
@@ -272,11 +272,11 @@ namespace OpenMS
       contaminants_.clear();
       const UInt COLS_EXPECTED = 8;
       Size line_number = 1;
-      for (TextFile::ConstIterator tf_it = tf.begin(); tf_it != tf.end(); ++tf_it, ++line_number)
+      for (const String& txt : tf)
```

Comment:
this loop is wrong... note the continue statement

Reply:
Similar to the above, can you expand on the issue with the continue